modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -125,13 +125,17 @@ This is a beta feature for v24.2 and is not recommended for use for production c
 ```bash
 curl -X PUT -d '{"state": "active"}' http://127.0.0.1:9644/v1/features/node_local_core_assignment
 ```
+
+xref:24.3@ROOT:upgrade:index.adoc[Upgrade to 24.3 or later] to use this feature for production clusters.
 ====
 
 In Redpanda, every partition replica is assigned to a CPU core on a broker. While Redpanda's default <<partition-replica-balancing,partition balancing>> monitors cluster-level events, such as the addition of new brokers or broker failure to balance partition assignments, it does not account for the distribution of partitions _within_ an individual broker. 
 
-Prior to Redpanda version 24.2, this meant that some cores on a broker could inadvertently host many partitions of heavily-used topics and cause the CPU to be xref:manage:monitoring.adoc#cpu-usage[overburdened]. Additionally, when the partition rebalance moved some partitions away from a broker, the remaining partitions did not necessarily rebalance across the broker's cores. Or, if a broker's core count was increased, Redpanda did not assign any partitions to the new cores until new partitions were created or old partitions were moved out.
+Prior to this version, this meant that some cores on a broker could inadvertently host many partitions of heavily-used topics and cause the CPU to be xref:manage:monitoring.adoc#cpu-usage[overburdened]. Additionally, when the partition rebalance moved some partitions away from a broker, the remaining partitions did not necessarily rebalance across the broker's cores. Or, if a broker's core count was increased, Redpanda did not assign any partitions to the new cores until new partitions were created or old partitions were moved out.
```

Comment:
I think we should remove this. It’s something that should have been explained in release notes rather than being part of the doc. Users don’t typically care about what a previous version used to do unless they’re reading release notes.